helm/values-dev.yaml

```diff
@@ -0,0 +1,73 @@
+microservice-chart:
+  namespace: gps
+  nameOverride: ""
+  fullnameOverride: ""
+  image:
+    repository: pagopadcommonacr.azurecr.io/pagopareportingorgsenrollment
+    tag: "0.0.1"
+    pullPolicy: Always
+  livenessProbe:
+    httpGet:
+      path: /actuator/health/liveness
+      port: 8080
+    initialDelaySeconds: 90
+    failureThreshold: 6
+    periodSeconds: 10
+  readinessProbe:
+    httpGet:
+      path: /actuator/health/readiness
+      port: 8080
+    initialDelaySeconds: 90
+    failureThreshold: 6
+    periodSeconds: 10
+  deployment:
+    create: true
+  service:
+    create: true
+    type: ClusterIP
+    port: 8080
+  ingress:
+    create: true
+    host: "your.host" # TODO set the host
+    path: /your-path-here/(.*) # TODO set your path
+  serviceAccount:
+    create: false
+    annotations: {}
+    name: ""
+  podAnnotations: {}
+  podSecurityContext:
+    seccompProfile:
+      type: RuntimeDefault
+  securityContext:
+    allowPrivilegeEscalation: false
+  resources:
+    requests:
+      memory: "512Mi"
+      cpu: "0.25"
+    limits:
+      memory: "512Mi"
+      cpu: "0.25"
+  autoscaling:
+    enable: true
+    minReplica: 3
+    maxReplica: 10
+    pollingInterval: 10 # seconds
+    cooldownPeriod: 50 # seconds
+    triggers:
+      - type: cpu
+        metadata:
+          # Required
+          type: Utilization # Allowed types are 'Utilization' or 'AverageValue'
+          value: "75"
+  envConfig:
+    FLOW_SA_CONNECTION_STRING: DefaultEndpointsProtocol=https;AccountName=pagopadflowsa;AccountKey=z92Nmp7swKo8NZdZnNyljpeLmxJDqbJ+hjZL7v8nYKHtTFi/IGV2sxXXx7Af/6xzml7deJiPAcCPXza/X3NGzg==;EndpointSuffix=core.windows.net
```

Comment:
Add  `WEBSITE_SITE_NAME` var. Also in UAT and PROD

Reply:
I don't know how to enhance the field